src/screens/Auth/LoginPage/LoginPage.tsx

```
@@ -1005,13 +1005,14 @@ const LoginPage = (): JSX.Element => {
                             >
                               <div
                                 ref={ref}
-                                className={`${className ?? ''} ${styles.selectOrgText}`}
+                                className={`${className ?? ''} ${styles.selectOrgText} flex align-items-center position-relative `}
                                 onMouseDown={onMouseDown}
                               >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)